src/app/shared/popover/popover.component.ts

```diff
@@ -177,4 +181,8 @@ export class PopoverComponent implements OnInit {
     this.router.navigateByUrl('/change-password')
   }
 
+  addTkProduct() {
+    this.dismissPopover();
+    this.navCtrl.navigateRoot('admin/add-tk-product');
```

Comment:
we can't use navigate root here because we can go back use router navigate method